doc/user/content/sql/show-create-connection.md

```diff
@@ -11,7 +11,7 @@ menu:
 ## Syntax
 
 ```sql
-SHOW CREATE CONNECTION <connection_name>
+SHOW [REDACTED] CREATE CONNECTION <connection_name>
```

Comment:
Should we add a blurb on what [REDACTED] does?
We can single source that statement.

Reply:
Added a patch with the description of the `REDACTED` option.  Feel free to update the description in the show_create_redacted_option.yml

Reply:
Great, thank you very much @kay-kim !